client.js

```diff
@@ -192,23 +524,53 @@ function pingUser (userID, seller, item, price, server, fullAuction) {
                             break;
                         case '🔕':
                             // Ignore this seller's auctions for this watch
+                            db.blockSeller(user.id, seller, null, watchId)
+                            user.send(`Let's cut out the noise!  No longer notifying you about auctions from ${seller} with regard to this watch.\n  To block ${seller} on all present and future watches, use \`\`!add block: ${seller}`);
+                            break;
+                        case '♻': //extend watch
+                            db.extendWatch(watchId)
+                            user.send(`Good things come to those who wait.  I added another 7 days to your \`\`${formattedItem}\`\` watch.`);
+                            break;
+                        default:
+                            break;
+                    }
+                })
+                collector.on('remove', (reaction, user) => {
+                    if (user.bot) return;
+                    switch (reaction.emoji.name) {
+                        case '💤':
+                            // unsnooze watch
+                            db.unsnooze('watch', watch.id);
+                            user.send(`Rise and grind.  No longer snoozing on your \`\`${item}\`\` watch on \`\`${server}\`\`.`).catch(console.error);
+                            break;
+                        case '❌':
+                            // renew this watch
+                            // db.endWatch(user.id, item, server);
+                            user.send(`Got it! No longer watching auctions for ${item} on P1999 ${server} server.`);
+                            break;
+                        case '🔕':
+                            // unblock the seller for this auction
+                            db.unblockSeller(user.id, seller, null, watchId)
+                            user.send(`Let's cut out the noise!  No longer notifying you about auctions from ${seller} with regard to this watch.\n  To block ${seller} on all present and future watches, use \`\`!add block: ${seller}`);
```

Comment:
Same as above, but should go away if combined.

Reply:
👍